### PR TITLE
HTML - Forms- Addy Abid

### PR DESCRIPTION
**Pull request by AddyAbid**
Screen Shot 2021-08-17 at 1 08 01 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label>
```
Comment (codingcodymiller) [inline html-forms/index.html:10]: This `label` isn't configured correctly to match with the form control element it is paired with. The `label` and the form control element it is associated with both need a specific attribute with a matching value. The attributes are mentioned in the instructions and the documentation, please re-read them and try to find the way to properly configure a `label` element.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          Email
+        </label>
+      </div>
+      <input type="email" name="email">
+      <div>
+        <label for="coding is cool">Why is coding cool?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:16]: There should not be spaces in the value here, you should use `kebab-case`. Spaces in attribute values generally means multiple values are being assigned to that attribute.

Comment (codingcodymiller) [reply html-forms/index.html:16]: Also, it is not configured correctly, just like the previous `label`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <div>
+        Favorite Language (choose one)
+      </div>
+      <input type="radio" name="language" value="HTML"/> HTML
+      <input type="radio" name="language" value="CSS" /> CSS
+      <input type="radio" name="language" value="Javascript" /> Javascript
```
Comment (codingcodymiller) [inline html-forms/index.html:29]: The text for each of these inputs should be a `label` element that is properly configured to match with the `input`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      <div>
+        Would you like to sign up for more information?
+      </div>
+      <input type="checkbox" name="sign up" value="Sign me up!"/> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:33]: "Sign me up!" should be a `label` element that is properly configured to match with the `input`

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      <div>
+        Comments
+      </div>
```
Comment (codingcodymiller) [inline html-forms/index.html:36]: This should be a `label`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good start here!! A few notes for you:

**Comment by AddyAbid** [comment]
Screen Shot 2021-08-17 at 7 24 45 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**